From: Sebastian Ramacher <sramacher@debian.org>
Date: Sun, 12 Nov 2017 18:29:02 +0100
Subject: va_drm: dlopen correct version of libva-x11

Signed-off-by: Sebastian Ramacher <sramacher@debian.org>
---
 va/drm/va_drm_auth_x11.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/va/drm/va_drm_auth_x11.c b/va/drm/va_drm_auth_x11.c
index 54e7402..78bc184 100644
--- a/va/drm/va_drm_auth_x11.c
+++ b/va/drm/va_drm_auth_x11.c
@@ -28,7 +28,7 @@
 #include <X11/Xlib.h>
 #include "va_drm_auth_x11.h"
 
-#define LIBVA_MAJOR_VERSION 1
+#define LIBVA_MAJOR_VERSION 2
 
 typedef struct drm_auth_x11             DRMAuthX11;
 typedef struct drm_auth_x11_vtable      DRMAuthX11VTable;
